credentialProvider/src/main/java/org/frankframework/credentialprovider/CredentialFactory.java

```diff
@@ -13,16 +13,15 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 */
-package nl.nn.credentialprovider;
+package org.frankframework.credentialprovider;
 
 import java.util.Collection;
 import java.util.function.Supplier;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
 import org.apache.commons.lang3.StringUtils;
-
-import nl.nn.credentialprovider.util.CredentialConstants;
+import org.frankframework.credentialprovider.util.CredentialConstants;
 
 public class CredentialFactory {
 	protected Logger log = Logger.getLogger(this.getClass().getCanonicalName());
```

Comment:
Deze moet eigenlijk ook nog oude namen aankunnen...
Willen we dit al overzetten, dit is eigenlijk een los ding, en zit niet verwoven in het framework